content/docs/start/experiments.md

```diff
@@ -25,6 +25,11 @@ $ dvc run -n evaluate \
 
 ### 💡 Expand to see what happens under the hood.
 
+The `-M` flag here specifies a metrics file, while the `--plots-no-cache` flag
+specifies a plot metrics file produced by this stage that will not tracked by
```

Comment:
will not _be_ ?

Reply:
they will tracked by DVC to some extent, better to say that they won't be saved into DVC cache.

Reply:
@utkarshsingh99, to summarize, please:
- Let's call them "options" not "flags" please.
- plot metrics file -> plots file
- that will not tracked by -> `that will not be <abbr>cached</abbr> by`
- Remove the last extra sentences about caching "You can choose... See dvc run..."
- Make "DVC generates a new stage..." (currently a separate paragraph) the last sentence in this same paragraph.